pynance/data/retrieve.py

```diff
@@ -10,14 +10,14 @@
 Wraps `Pandas Remote Data Access 
 <http://pandas.pydata.org/pandas-docs/stable/remote_data.html>`_.
 """
-
+from datetime import datetime
 from ftplib import FTP
 from functools import partial
 import io
 
 import pandas_datareader.data as web
 
-def get(equity, start, end):
+def get(equity, start, end = datetime.today().date().isoformat()):
```

Comment:
Another option worth considering: Change the signature to `get(equity, *args)` and have the body be:

    return web.DataReader(equity, 'yahoo', *args)

I think that would work, and it's clearly simplest. Then we could also show in the documentation that start defaults to None.